### PR TITLE
Add sphinx docs

### PR DESCRIPTION
**Pull request by martamatos**
Work in progress, just to make the discussion easier.

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'yaml2sbml'
+copyright = '2021'
+author = 'jakob, yannik, marta, dilan, etc. [TODO]'
```
Comment (jvanhoefer) [inline doc/conf.py:22]:
```suggestion
author = 'Jakob Vanhoefer, Marta R. A. Matos, Dilan Pathirana, Yannik Schälte'
```

File: doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Welcome to yaml2sbml's documentation!
+===============================================
+
+
+This small package allows the user to [TODO]
```
Comment (jvanhoefer) [inline doc/index.rst:5]: I will fix this

**Review by jvanhoefer — APPROVED** [review]

File: doc/index.rst (hunk 2 above)
Comment (yannikschaelte) [inline doc/index.rst:5]: maybe use the README.md here?

File: doc/install.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
+Installation
+===============================================
+
+
+To install go to the main folder and do:
+
+```pip install .```
+
+
+Requirements for users
```
Comment (yannikschaelte) [inline doc/install.rst:10]: I would rather not list the requirements here, as this is likely to get outdated. I.e. requirements only listed in setup.py. / requirements.txt / setup.cfg / wherever it is.

File: doc/install.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+---------------------------
+
+ - Python 3.6+
+ - python-libsbml>=5.18.0
+ - PyYAML>=5.1
+ - pandas>=1.0.1
+ - petab>=0.1.4
+
+Extras (to run examples):
```
Comment (yannikschaelte) [inline doc/install.rst:19]: if not done yet, I would turn this into an "extras_require".

Comment (jvanhoefer) [reply doc/install.rst:19]: Should already be in there... :)

**Comment by jvanhoefer** [comment]
Open points
- [ ] Jupiter notebook
- [ ]  Intro text

**Comment by martamatos** [comment]
Just to say that i've figured out how to include jupyter notebooks and will push tonight :)

**Comment by jvanhoefer** [comment]
@martamatos Then I wait for you to push, before doing "my stuff"

File: doc/conf.py (hunk 1 above)
Comment (martamatos) [reply doc/conf.py]: Addressed :)

File: doc/index.rst (hunk 2 above)
Comment (martamatos) [reply doc/index.rst]: I took the liberty of adding some text, but feel free to change it all (i just think it's easier to change something than to write from scratch) :)

File: doc/install.rst (hunk 3 above)
Comment (martamatos) [reply doc/install.rst]: Makes sense :)
Just added a tiny instruction subsection to the main page and removed the install page.

**Comment by martamatos** [comment]
Okidoki, i think this is ready from my side, which doesn't mean it really is ready, so feedback is super welcome, or feel free to modify things directly, whatever you prefer :)

You can see the documentation here: https://yaml2sbml.readthedocs.io/en/latest/, and it's probably a good idea to use a private window to open the link, as sometimes i can't see the latest changes in a normal window (due to cache probably).

**Comment by martamatos** [comment]
Aaaand, i accidently removed the examples folder from inside the doc folder (i thought i had put it there by mistake) :/

However, the `doc` folder might not be the best place to have the `examples` folder, since now there are a bunch of other files there too.

So my suggestion is to either put the sphinx docs in a different folder (`sphinx_docs`?), or put the `examples` folder in the main directory.  Ideas?

(and yes, i know there is a conflict between the branches, which i'll fix as soon as we decide on this)

**Comment by yannikschaelte** [comment]
> Aaaand, i accidently removed the examples folder from inside the doc folder (i thought i had put it there by mistake) :/
> 
> However, the `doc` folder might not be the best place to have the `examples` folder, since now there are a bunch of other files there too.
> 
> So my suggestion is to either put the sphinx docs in a different folder (`sphinx_docs`?), or put the `examples` folder in the main directory. Ideas?
> 
> (and yes, i know there is a conflict between the branches, which i'll fix as soon as we decide on this)

I think examples/ in the base folder makes sense.

**Review by yannikschaelte — APPROVED** [review]
Looks good to me!

**Comment by jvanhoefer** [comment]
I will resolve the merge conflicts and include the new notebooks, that are new in the master branch and not in the toctree yet... :)

After that, looks ready to merge for me :)

**Pull request merged**